components/tidb_query_datatype/src/codec/data_type/bit_vec.rs

```diff
@@ -32,8 +32,6 @@ impl BitVec {
         self.length += 1;
         if value {
             self.data[idx] |= mask;
```

Comment:
@breeswish What about setting all bits to 1 on initialization? Is this optimization worth?

Reply:
According to the benchmark report, latency has been increased a bit?

Reply:
I think that's fluctuation in benchmark.